recipe/meta.yaml

```diff
@@ -38,7 +37,6 @@ test:
     - rope.refactor.importutils
   requires:
     - pip
-    - python <3.10
```

Comment:
to remove this line here is a bit too early AFAICS

Reply:
Fixed!